test/main.js

```diff
@@ -64,9 +65,15 @@ test('event behavior with over-ridden `render` & `remove` fns', function (t) {
         ++renderCount;
         t.ok(true, 'view `render` event happened on `render()`');
     });
+    view.on('remove', function(view, value) {
+        ++removeCount;
+    });
     view.render();
+    t.equal(removeCount, 0, '`remove` event not called, pre- or post- render()');
+    t.equal(renderCount, 1, '`render` triggered exactly once, post- render()');
     view.remove();
-    t.equal(1, renderCount, 'over-ridden `render` triggered render just once');
+    t.equal(removeCount, 1, '`remove` triggered exactly once, post- remove()');
+    t.equal(renderCount, 1, '`render` triggered exactly once, post- remove()');
```

Comment:
I'd change this one to `render not triggered post- remove()` as it's getting kinda confusing :P